jet.h

```diff
@@ -18,13 +68,44 @@ class JetCorrProvider : public CorrectionsBase<JetCorrProvider> {
         };
         return names.at(scale);
     }
+    static const int GetScaleIdx(UncScale scale){
+        static const std::map<UncScale, int> scale_indexes = {
+            {UncScale::Up, 1},
+            {UncScale::Down,2},
+            {UncScale::Central,0},
+        };
+        return scale_indexes.at(scale);
+    }
     static bool sourceApplies(UncSource source)
```

Comment:
this function is not relevant -> remove

Reply:
I need it for how the variation are emplaced in the map (in JMESystematicCalculators)

Reply:
but it should be always `true`. in which case it can be `false`?

Reply:
it is not a boolean. It represents the index of the variable https://gitlab.cern.ch/cp3-cms/CMSJMECalculators/-/blob/main/src/JMESystematicsCalculators.cc#L306

since we evaluate the correction for each uncertainty, the value 0 represents the central (nominal), the value 1 should be the up variation and the value 2 the down variation (I already changed it in the current code - not yet pushed)